requester/requester.go

```diff
@@ -54,10 +61,11 @@ func (r *Requester) Run() Report {
 // response body, invalidating the entire response, as it is not a remote
 // server error.
 type Record struct {
-	Time  time.Duration `json:"time"`
-	Code  int           `json:"code"`
-	Bytes int           `json:"bytes"`
-	Error error         `json:"error"`
+	Time   time.Duration `json:"time"`
+	Code   int           `json:"code"`
+	Bytes  int           `json:"bytes"`
+	Error  error         `json:"error"`
```

Comment:
Not really in the scope of this PR but could be added because we could avoid to open a new one to do so:
Error could be omitted if empty in json.
```suggestion
	Error  error         `json:"error,omitempty"`
```